CefSharp.Core/ManagedCefBrowserAdapter.h

```diff
@@ -76,12 +76,12 @@ namespace CefSharp
             if (offScreenRendering)
             {
                 _clientAdapter = new RenderClientAdapter(webBrowserInternal,
-                    gcnew Action<int>(this, &ManagedCefBrowserAdapter::OnAfterBrowserCreated));
+                    gcnew Action<int>(this, &ManagedCefBrowserAdapter::OnAfterBrowserCreated), this);
```

Comment:
We should ditch the `Action` and add a method to the interface now that `ClientAdapter` has a reference to `ManagedCefBrowserAdapter`.